tests/main/snapctl/task.yaml

```diff
@@ -0,0 +1,80 @@
+summary: Check that `snapctl` can be run from within hooks
+
+systems: [-ubuntu-core-16-64]
+
+prepare: |
+    echo "Ensure we have a core snap with snap run"
+    $SPREAD_PATH/tests/lib/reset.sh
+    snap install --channel=candidate ubuntu-core
```

Comment:
Ah, thank you. A few other tests can probably be fixed as well (all the snap run ones, basically).